packages/style-engine/class-wp-style-engine-css-rule.php

```diff
@@ -32,16 +32,27 @@ class WP_Style_Engine_CSS_Rule {
 		 */
 		protected $declarations;
 
+		/**
+		 * The @rule.
+		 *
+		 * @var string
+		 */
+		protected $at_rule;
+
+
 		/**
 		 * Constructor
 		 *
 		 * @param string                                    $selector     The CSS selector.
 		 * @param string[]|WP_Style_Engine_CSS_Declarations $declarations An associative array of CSS definitions, e.g., array( "$property" => "$value", "$property" => "$value" ),
 		 *                                                                or a WP_Style_Engine_CSS_Declarations object.
+		 * @param string                                    $at_rule        A CSS @rule.
```

Comment:
Think it would be good to add an example here. It's not clear to me from reading it if I'm supposed to provide e.g. the `@` symbol.